### DISCUSSION TITLE
Filtering groups

### DISCUSSION THREAD

**Discussion by sebekhtc** [zwave-js/zwave-js-ui#3339] [Ideas]
I just found I can filter my devices and then group it. 
1. I Can't find how to make another group without deleting the existing. If there is no way to create another group can you add that feature?
2. Could you add a feature to save groups so they can be used in the future?
Regards!

**Comment by robertsLando** [comment]
@sebekhtc I'm sorry but there is actually no support for such feature on lower side framework and implementing it would require too much effort, if this issue wll get more attention by others I could consider it in the future.

About saving groups that could be something that I could consider adding when restoring columns filters from localstorage

**Reply by sebekhtc** [reply]
thanks, saving would be also good to recall such groups (so point 1 would somehow  be solved this way)

**Comment by sebekhtc** [comment]
closing the discussion...

**Reply by sebekhtc** [reply]
I feel embarassed now,  I never used groups without entering nothing to filter. Leaving  filter fileds empty the feature groups already nicely eg. by location. That is enough for me. Apologies for taking your time.

**Reply by robertsLando** [reply]
No worries, glad you found a solution :)
